routers/api/v1/repo/pull.go

```diff
@@ -405,7 +405,7 @@ func EditPullRequest(ctx *context.APIContext, form api.EditPullRequestOption) {
 		}
 	}
 
-	if ctx.Repo.IsWriter() && (form.Labels != nil && len(form.Labels) > 0) {
+	if ctx.Repo.IsWriter() && form.Labels != nil {
```

Comment:
Will this remove all labels not provided in the request?

Example: I have 5 labels.
1. Request: Add label 1,4,2
2. Request: Add label 3

Would this then have 1,2,3,4 or only 3?

Reply:
Yes, this will remove all labels not provided in the request.
This would only have the label3 then.